nixos/modules/misc/ids.nix

```diff
@@ -288,7 +288,7 @@ in
       telegraf = 256;
       gitlab-runner = 257;
       postgrey = 258;
-      hound = 259;
+      # hound = 259; # unused, removed 2023-11-21
```

Comment:
For things like backup and restore, it's generally useful to have a stable UID between machines. Otherwise, use `DynamicUser = true` so that the UIDs fix themselves automatically.

Reply:
DynamicUser doesn't work that well for services that store some amount of data. My hound is 4.3GB big and chowning that on every startup would take a second or two.

Also I don't think people are backing up their hound index. It is just a search index which gets recreated if it doesn't exist, so there is no data worth backing up and I have excluded it from my backups.